docs/quickstart/junit_5_quickstart.md

```diff
@@ -0,0 +1,110 @@
+# JUnit 5 Quickstart
+
+It's easy to add Testcontainers to your project - let's walk through a quick example to see how.
+
+Let's imagine we have a simple program that has a dependency on Redis, and we want to add some tests for it.
+In our imaginary program, there is a `RedisBackedCache` class which stores data in Redis.
+ 
+You can see an example test that could have been written for it (without using Testcontainers):
+
+<!--codeinclude-->
+[Pre-Testcontainers test code](../examples/junit5/redis/src/test/java/quickstart/RedisBackedCacheIntTestStep0.java) block:RedisBackedCacheIntTestStep0
+<!--/codeinclude-->
+
+Notice that the existing test has a problem - it's relying on a local installation of Redis, which is a red flag for test reliability.
+This may work if we were sure that every developer and CI machine had Redis installed, but would fail otherwise.
+We might also have problems if we attempted to run tests in parallel, such as state bleeding between tests, or port clashes.
+
+Let's start from here, and see how to improve the test with Testcontainers:  
+
+## 1. Add Testcontainers as a test-scoped dependency
+
+First, add Testcontainers as a dependency as follows:
+
+```groovy tab='Gradle'
+def junitJupiterVersion = '5.4.2'
```

Comment:
If these could be included from the real `build.gradle` file, that would be great.

Reply:
I don't think it will be possible as the code example currently depends on a project dependency of TestContainers. It would have to be a mix of external code include and plain-text code.

The JUnit 4 page currently does the same thing: https://github.com/testcontainers/testcontainers-java/blob/master/docs/quickstart/junit_4_quickstart.md. This is one of the reasons why I proposed to have self-contained code example in https://github.com/testcontainers/testcontainers-java/issues/1193 as it would reflect how end-user write their projects.